spec/features/basic_user_pending_request_spec.rb

```diff
@@ -4,30 +4,15 @@
 
 RSpec.feature 'Publish Record', type: :feature do
   before do
-    stub_request(:get, 'https://country.beta.openregister.org/record/zz.tsv')
-      .with(headers: { 'Accept' => '*/*', 'Accept-Encoding' => 'gzip, deflate', 'Host' => 'country.beta.openregister.org' })
+    stub_request(:get, 'https://country.register.gov.uk/record/zz.tsv')
```

Comment:
Does this stub need to be here? I'm not sure we use this endpoint now that we've switched over to the new client library (which doesn't use `tsv`).